src/utils/compileJsx.ts

```diff
@@ -1,11 +1,22 @@
 import { transform } from '@babel/standalone';
 
-export const openFragmentTag = '<React.Fragment>';
-export const closeFragmentTag = '</React.Fragment>';
+export const ReactFragmentPragma = 'R_F';
+export const ReactCreateElementPragma = 'R_cE';
```

Comment:
I feel like we could get away with `RF` and `RcE`.

Reply:
After careful consideration, I think these have a lower chance to collide with anything in scope. I've added a guard here https://github.com/seek-oss/playroom/pull/304/commits/082ec1822d065ffc08292edc78fe0e670c592fc8